src/lib/components/Modals/LoginModal.tsx

```diff
@@ -0,0 +1,196 @@
+import React from 'react';
+import { useForm, DeepMap, FieldError } from 'react-hook-form';
+import {
+  Button,
+  TextField,
+  Grid,
+  Typography,
+  Link,
+  FormControlLabel,
+  Checkbox,
+  Dialog,
+  DialogTitle,
+  DialogContent,
+} from '@material-ui/core';
+import { RegistrationModal } from './RegistrationModal';
+
+export interface IInputs {
+  email: string;
+  password: string;
+}
+
+export const LoginModal: React.FC = () => {
+  const [loginOpen, setLoginOpen] = React.useState(false);
+  const [registrationOpen, setRegistrationOpen] = React.useState(false);
+
+  // eslint-disable-next-line @typescript-eslint/unbound-method
+  const { register, handleSubmit, errors } = useForm<IInputs>();
+
+  const handleLoginOpen = () => {
+    setLoginOpen(true);
+  };
+
+  const handleLoginClose = () => {
+    setLoginOpen(false);
+  };
```

Comment:
you can avoid using such methods.. 
just use `setLoginOpen(true)` when you need that, no need for wraping this :)